packages/ibm-products/src/components/Coachmark/utils/enums.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable max-len */
+
+/**
+ * Reserved for future expansion, i.e. "RING".
+ */
+export const BEACON_KIND = {
+  DEFAULT: 'default',
+  //RING: 'ring',
```

Comment:
Remove if no longer needed

Reply:
Yep, it was a future thing, will remove as it's an unknown at this point.